### ISSUE TITLE
About data loading

### ISSUE THREAD

**Issue by ARnnn** [DVLP-CMATERJU/RectiNet#4]
I I have downloaded the data you provided, but in the dataset.py In, label is. Npz format, but I didn't find this format in the data you provided. Can you tell me how to apply the downloaded data to the code you provided?

**Comment by hmrishavbandy** [comment]
Please note: You have to generate your own dataset. The data generation code can be found [here](https://github.com/XiyanLiu/AGUN). Use this code to generate your own data. The ground truth of this data would be of .npz format.

**Comment by ARnnn** [comment]
Does the ground truth correspond to an image without distortion

**Comment by hmrishavbandy** [comment]
No, the ground truth does not correspond to the scanned image. The ground truth is a dense-grid containing x and y co-ordinates for mapping the warped image to a dewarped image. For dewarping with the ground truth, we make use of `torch.nn.functional.grid_sample()`

**Comment by ARnnn** [comment]
In the data generation code provided by you, we finally get the data in  .pkl format. Each   .pkl contains a distorted image, a ground truth grid and a ground truth image. Whether the ground truth grid is the  .npz data that needs to be generated

> 不，地面真相与扫描的图像不符。地面真相是一个密集的网格，包含 x 和 y 坐标，用于将扭曲的图像映射到去扭曲的图像。对于与地面真理的解战，我们利用`torch.nn.functional.grid_sample()`

  In the data generation code provided by you, we finally get the data in  .pkl format. Each   .pkl contains a distorted image, a ground truth grid and a ground truth image. Whether the ground truth grid is the  .npz data that needs to be generated

**Comment by hmrishavbandy** [comment]
Yes the ground truth would be in an NPZ format
